Unithereum.CodeGen/Config.cs

```diff
@@ -0,0 +1,279 @@
+            "continue", "decimal", "default", "delegate", "do", "double", "else", "enum", "event", "explicit", "extern",
+            "false", "finally", "fixed", "float", "for", "foreach", "goto", "if", "implicit", "in", "int", "interface",
+            "internal", "is", "lock", "long", "namespace", "new", "null", "object", "operator", "out", "override",
+            "params", "private", "protected", "public", "readonly", "ref", "return", "sbyte", "sealed", "short",
+            "sizeof", "stackalloc", "static", "string", "struct", "switch", "this", "throw", "true", "try", "typeof",
+            "uint", "ulong", "unchecked", "unsafe", "ushort", "using", "virtual", "void", "volatile", "while"
+        };
+
+        /// <summary>
+        ///     Absolute path to `dotnet` executable.
+        /// </summary>
+        public string? DotnetPath { get; }
+
+        /// <summary>
+        ///     Namespace prefix for generated contract service classes.
+        ///     <para>Default: (Application.productName).ContractServices</para>
+        /// </summary>
+        public string NsPrefix { get; }
+
+        /// <summary>
+        ///     Relative path to `Asset` directory for generated contract service class files.
+        ///     <para>Default: ContractServices</para>
+        /// </summary>
+        public string OutputDir { get; }
+
+        [JsonConstructor]
+        public Config([Optional] string? dotnetPath, [Optional] string? nsPrefix, [Optional] string? outputDir)
+        {
+            const string defaultName = "ContractServices";
+
+            if (nsPrefix != null && !ValidateNamespacePrefix(nsPrefix))
+            {
+                var error = new InvalidOperationException(
+                    $"Invalid Unithereum CodeGen config: NsPrefix ({nsPrefix}). Use proper C# namespace identifier.");
+                Debug.LogError(error.Message);
+                throw error;
+            }
+
+            DotnetPath = dotnetPath ?? GetDotnetPath();
+            NsPrefix = nsPrefix ?? GetNamespacePrefix() + '.' + defaultName;
+            OutputDir = outputDir ?? defaultName;
+        }
+
+        public static Config GetConfig()
+        {
+            var path = Path.Combine(Path.GetDirectoryName(Application.dataPath)!, "codegen.config.json");
+            return File.Exists(path) ? ReadFromJsonFile(path) : new Config();
+        }
+
+        private static Config ReadFromJsonFile(string configPath)
+        {
+            var json = File.ReadAllText(configPath);
+            var config = JsonConvert.DeserializeObject<Config>(json, new ConfigDeserializer());
+            if (config is null) throw new InvalidOperationException("Failed to create config object.");
+            return config;
+        }
+
+        public override string ToString()
+        {
+            return base.ToString() + ": " + JsonConvert.SerializeObject(this);
+        }
```

Comment:
```suggestion
        public override string ToString() => JsonConvert.SerializeObject(this);
```

I think when people expect to use `ToString()` on this class, they might expect to use the result string as-is programmatically instead of displaying it for human reading directly.